featurebyte/service/feature_list.py

```diff
@@ -303,6 +306,40 @@ async def create_document(self, data: FeatureListServiceCreate) -> FeatureListMo
             await self._update_features(document.feature_ids, inserted_feature_list_id=insert_id)
         return await self.get_document(document_id=insert_id)
 
+    async def list_documents_iterator(  # type: ignore[override]
```

Comment:
should we deprecate this iterator helper entirely? seems like there are only 20+ usages left, and we continue to make changes like this pretty often. that way we can reduce the mental overhead of having to choose between which iterator to use.

it is a bit unfortunate that we lose the type checking though since we only get dictionaries back.

Reply:
So far it should only be an issue for `feature`, `target` & `feature_list` as their object contains graph. Other asset types should be ok. Wdyt?